### ISSUE TITLE
执行命令tinkerPatchRelease时编译不通过，assembleRelease是正常的。

### ISSUE THREAD

**Issue by yqyzxd** [Tencent/tinker#454]
异常类型：tinkerPatchRelease编译异常

手机型号：

手机系统版本：

tinker版本：如:1.7.7

gradle版本：如:2.3.0

是否使用热更新SDK：TinkerPatch SDK

系统：如:Mac

堆栈/日志：
tinkerpatch change application name from com.xkd.dinner.App to com.tinkerpatch.sdk.loader.TinkerPatchReflectApplication
:app:tinkerProcessReleaseManifest
tinker add tinker_id_1.0.0 to your AndroidManifest.xml /Users/wind/StudioProjects/Dinner/app/build/intermediates/manifests/full/release/AndroidManifest.xml
tinker add com.tinkerpatch.sdk.loader.TinkerPatchReflectApplication to dex loader pattern
tinker add com.tencent.tinker.loader.* to dex loader pattern
tinker gen AndroidManifest.xml in build/intermediates/tinker_intermediates/AndroidManifest.xml
:app:tinkerProcessReleaseResourceId
we build app apk with apply resource mapping file /Users/wind/StudioProjects/Dinner/app/build/bakApk/app-1.0.0-0425-11-17-21/release/app-release-R.txt
tinker gen resource public.xml in build/intermediates/tinker_intermediates/public.xml
tinker gen resource idx.xml in build/intermediates/tinker_intermediates/idx.xml
:app:processReleaseResources
/Users/wind/StudioProjects/Dinner/app/build/intermediates/res/merged/release/values/public.xml:484: AAPT: Public symbol attr/qupai_originalHeight declared here is not defined.
    
/Users/wind/StudioProjects/Dinner/app/build/intermediates/res/merged/release/values/public.xml:271: AAPT: Public symbol attr/qupai_originalWidth declared here is not defined.
    
/Users/wind/StudioProjects/Dinner/app/build/intermediates/res/merged/release/values/public.xml:484: error: Public symbol attr/qupai_originalHeight declared here is not defined.
/Users/wind/StudioProjects/Dinner/app/build/intermediates/res/merged/release/values/public.xml:271: error: Public symbol attr/qupai_originalWidth declared here is not defined.

:app:processReleaseResources FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processReleaseResources'.
> com.android.ide.common.process.ProcessException: Failed to execute aapt

引入了趣拍短视频库。

**Comment by shwenzhang** [comment]
/Users/wind/StudioProjects/Dinner/app/build/intermediates/res/merged/release/values/public.xml:484: error: Public symbol attr/qupai_originalHeight declared here is not defined.
/Users/wind/StudioProjects/Dinner/app/build/intermediates/res/merged/release/values/public.xml:271: error: Public symbol attr/qupai_originalWidth declared here is not defined.

**Comment by yqyzxd** [comment]
在public.xml文件中的值
<public type="attr" name="qupai_originalHeight" id="0x7f010105" />
<public type="attr" name="qupai_originalWidth" id="0x7f010104" />
android中可以引用到这个值
 R.styleable.qupai_AspectRatioLayout_qupai_originalHeight

**Comment by yqyzxd** [comment]
在public.xml文件中的值
<public type="attr" name="qupai_originalHeight" id="0x7f010105" />
<public type="attr" name="qupai_originalWidth" id="0x7f010104" />
android中可以引用到这个值
R.styleable.qupai_AspectRatioLayout_qupai_originalHeight

**Comment by shwenzhang** [comment]
fixed with 1.7.9

**Comment by tysheng** [comment]
```
:app:processDevelopmentReleaseResourcesAGPBI: {"kind":"error","text":"Public symbol attr/qupai_originalHeight declared here is not defined.","sources":[{"file":"/Users/shengtianyang/Desktop/Eforums2/app/build/intermediates/data-binding-layout-out/development/release/values/public.xml","position":{"startLine":163}}],"original":"","tool":"AAPT"}
AGPBI: {"kind":"error","text":"Public symbol attr/qupai_originalWidth declared here is not defined.","sources":[{"file":"/Users/shengtianyang/Desktop/Eforums2/app/build/intermediates/data-binding-layout-out/development/release/values/public.xml","position":{"startLine":265}}],"original":"","tool":"AAPT"}
/Users/shengtianyang/Desktop/Eforums2/app/build/intermediates/data-binding-layout-out/development/release/values/public.xml:164: error: Public symbol attr/qupai_originalHeight declared here is not defined.
/Users/shengtianyang/Desktop/Eforums2/app/build/intermediates/data-binding-layout-out/development/release/values/public.xml:266: error: Public symbol attr/qupai_originalWidth declared here is not defined.

 FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processDevelopmentReleaseResources'.
> com.android.ide.common.process.ProcessException: Failed to execute aapt
```
bugly 1.3.0，也就是tinker 1.7.9，貌似还是有这个问题
@shwenzhang
